src/OpenTelemetry.Api/Trace/SpanExtensions.cs

```diff
@@ -158,45 +158,45 @@ public static TelemetrySpan PutHttpStatusCode(this TelemetrySpan span, int statu
         {
             span.PutHttpStatusCodeAttribute(statusCode);
 
-            var newStatus = Status.Ok;
+            Status newStatus;
```

Comment:
To realize what was the default for `Status` I had to peek into it, then into `CanonicalStatus`:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/f0a0b07ffbfa9d87a50017d6863c9b20195b2e6f/src/OpenTelemetry.Api/Trace/CanonicalCode.cs#L27

I understand the default value of the struct there would essentially be the same but I do prefer being explicit there.
It shows also intent. Unless there was no intent for it to default to `OK` as indicated by the `else { Status.Unknown }` part.

So perhaps start of with:
```suggestion
            Status newStatus = Status.Unknown;
```

And remove the last `else` block so unless a specific value is mapped, stays `unknown`.

Reply:
Yea, I've deleted default value because it was always changed, also by the `else` block. Not because equality with `default(Status)` :)

I like your idea, it's even shorter. Changed